docs/internal-documentation/templates/specifications/tests/Template01.md

```diff
@@ -0,0 +1,262 @@
+> > it. Dependency would mean that the test case is affected by the other test
+> > case, or even cannot be run if the other has not been run. Assuming that
+> > another test case has been run is just to make it possible to ignore certain
+> > errors. E.g.:
+
+It is assumed that *Child Zone* has been tested and reported by [Basic04]. This
+test case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+
+## Inputs
+
+> > Input data to the test case. Always include the Child zone, but it can also
+> > be other data units.
+> >
+> > The input data name is put in quote marks '""', then a hyphen '-' as a
+> > separator, and finally a description.
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+> > First we can have bullets, if applicable, that states noteable things about
+> > the execution or the messages. E.g.
+* If no CDS record is found, the test case will terminate early
+  with no message tag outputted.
+* If a CDS record is of "delete" type, then it can by definition not
+  match or point at any DNSKEY record.
+
+> > Here is a table of all message tags referred to in the steps. The tag in the
+> > first column, the default severity level in the second, and a statement on
+> > when the message is outputted in the third. Always use the same table set-up,
+> > but with the correct tags. E.g.:
+
+Message Tag outputted         | Level   | Arguments  | Description of when message tag is outputted
+:-----------------------------|:--------|:-----------|:--------------------------------------------
+T01_BROKEN_DNSSEC             | ERROR   | ns_ip_list | Replies do not follow the standard.
+T01_ALGO_NOT_SUPPORTED_BY_ZM  | NOTICE  | algo_descr, algo_num | The algorithm used is not supported by the Zonemaster implementation.
+T01_HAS_NSEC                  | INFO    |            | The *Child Zone* uses NSEC.
+T01_HAS_NSEC3                 | INFO    |            | The *Child Zone* uses NSEC3.
+T01_INCONSISTENT_DNSSEC       | ERROR   | keytag     | The configuration of the zone is inconsistent with respect to DNSSEC.
+
+The value in the Level column is the default severity level of the message. The
+severity level can be changed in the [Zonemaster-Engine profile]. Also see the
+[Severity Level Definitions] document.
+
+The argument names in the Arguments column lists the arguments used in the
+message. The argument names are defined in the [argument list].
+
+
+> > The message tags should be formed following the [Message Tag Specification].
+> >
+> > The default severity level is selected from the [Severity Level Definitions].
+
+## Test procedure
+
+> > This section contains the detailed steps to execute the test case. The steps
+> > should be as explicit as possible to avoid that different implementations or
+> > executions do different interpretations or assumptions.
+> >
+> > The steps should written in such a way that it reasonbly possible to use
```

Comment:
missing word: "should _be_ written"
typo: reasonbly -> reasonably
missing word: "that it _is_ reasonably"